textalyze.js

```diff
@@ -43,9 +43,21 @@ function stringToCharacters(text) {
  */
 function sanitize(text) {
   if (typeof text === 'string' || text instanceof String) {
-    return text.toLowerCase();
+    return text.toLowerCase().replace(/[^a-zA-Z]+/g, '');
   }
-  return text;
+  return null;
+}
+
+/**
+ * Prints text frequencies histogram
+ * @param {Map} frequencies
+ */
+function printHistogram(frequencies) {
+  frequencies.forEach((freq, value) => {
+    const percentage = `[${freq < 0.1 ? ' ' : ''}${(freq * 100).toFixed(2)}%]`;
+    const length = ''.padStart(Math.floor(freq / 0.005), '=');
```

Comment:
Here I'd use `String.prototype.repeat`, so:

```javascript
const maxWidth = 200;
'='.repeat(freq * maxWidth)
```

`String.prototype.repeat` does the flooring, i.e., `str.repeat(x) === str.repeat(Math.floor(x))` for any number `x`.

Reply:
A much better way to understand, thanks!!